### PR TITLE
Add CMake for tests. currently WIP not working

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,5 +48,4 @@ install:
     - cmake -D CMAKE_INSTALL_PREFIX=$HOME/miniconda -D CMAKE_INSTALL_LIBDIR=lib -D CMAKE_C_COMPILER=$CC -D CMAKE_CXX_COMPILER=$CXX ..
```
Comment (martinRenou) [inline .travis.yml:48]: I feel like you need to compile tests first

**Comment by marimeireles** [comment]
I'll merge this because I need the CMAKE file for building tests on travis and seeing them failing here.

**Comment by martinRenou** [comment]
No... You did not need to merge this. Why do you think you needed to merge it?

**Comment by marimeireles** [comment]
Oh no, because Travis said I didn't have a CMake inside my test directory.

**Comment by martinRenou** [comment]
And Travis was right. You were not compiling the tests, that is why the test executable was not there. The Travis build on master is now failing (which means that no other PR can pass the CI until we fixed master).

Let's see if we can fix it with your other PR.

**Comment by martinRenou** [comment]
This is not a big mistake right now (xeus-sqlite is not widely used, it has not been released yet, and only us are contributing right now), but it's generally a bad idea to merge PRs if the CI is failing.

**Pull request merged**